examples/full-example/.mendelrc

```diff
@@ -18,17 +18,17 @@ variation-config:
 
 route-config:
   # bundle does not have extension has it can be css bundle.
-  variation: /mendel/:variations/:bundle
-  hash: /mendel/:hash/:bundle
+  variation: /mendel/:variations/:bundle # Route pattern during development for variational assets
+  hash: /mendel/:hash/:bundle # Route pattern during production for variational assets
 
 transforms: # a list of all available transforms for all envs and types
   babel:
-    plugin: mendel-transform-babel
-    options:
+    plugin: mendel-transform-babel # Name of package for transformations
+    options: # Extra arguments to be passed to plugin
```

Comment:
I think we should document only the first plugin, to remove clutter, and here is a suggestion:
```yaml
  # plugins are declared upfront and used by "types" bellow
  # you can have multiple instances of the same plugin with different ids
  babel:                            # id or reference of a plugin instance
    plugin: mendel-transform-babel  # name of the plugin to be used, plugin must be installed in your project
    options:                        # configuration object passed to plugin instance, each plugin has it's own format and documentation
```